lib/gems/pending/appliance_console.rb

```diff
@@ -393,6 +391,8 @@ module ApplianceConsole
             File.delete(DB_RESTORE_FILE)
           elsif !rake_success
             say("\nDatabase restore failed")
+            connections = ApplianceConsole::Utilities.db_connections - 1
```

Comment:
No, the `db_connections` method actually connects to the database. So the `- 1` is for the one we are currently using to query.